style.css

```diff
@@ -576,3 +569,437 @@ footer {
+    display: flex;
+    gap: 0.5vw;
+    margin-top: 3vh;
+  }
+
+  .list {
+    flex: 1;
+  }
+
+  .image-hover {
+    z-index: 2;
+    height: 60vh;
+  }
+
+  .card:hover > .image-hover {
+    display: flex;
+    flex-direction: column;
+    height: 60vh;
+    width: 27vw;
+    margin: 0.5%;
+    padding: 0%;
+    cursor: pointer;
+  }
+
+  .card:hover > .container-card-heading,
+  .card:hover > .container-card-paragraph {
+    display: none;
+  }
+
+  .card:hover > .cardlist {
+    display: none;
+  }
+
+  .card:hover > .container-seeproject-button {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .button {
+    margin-top: 14vh;
+  }
+
+  /* --------------SECTION ABOUT ME-------------------------------------------------------- */
+
+  .aboutme {
+    display: flex;
+    background-image: url(img/image-contact-desktop.png);
+    background-position-y: 17%;
+    background-size: 40%;
+    margin-top: 8vh;
+    padding-bottom: 0;
+    flex: 1;
+    width: 100%;
+  }
+
+  .hello {
+    display: none;
+  }
+
+  .headeraboutme {
+    margin-left: 8vw;
+    flex: 1;
+  }
+
+  .heythere {
+    font-size: 20px;
+    line-height: 32px;
+    width: 43vw;
+    margin-left: 8vw;
+    flex: 1;
+  }
+
+  .aboutme div .button1 {
+    margin-bottom: 15vh;
+    margin-left: 8vw;
+    width: 14vw;
+    flex: 1;
+  }
+
+  .aboutme .container-divisionline2 .division {
+    width: 82%;
+    margin-left: -2%;
+    padding: 0;
+  }
+
+  .subcontainer-about {
+    display: grid;
+    grid-template-columns: 33% 33% 33%;
+    grid-template-rows: 10% 90%;
+    width: 87%;
+    margin-left: 10%;
+  }
+
+  .container-containerlanguagesquare {
+    justify-content: flex-start;
+  }
+
+  .container-containerlanguagesquare:first-child {
+    grid-area: 1/1/2/2;
+  }
+
+  .container-containerlanguagesquare:nth-child(3) {
+    grid-area: 1/2/2/3;
+  }
+
+  .container-containerlanguagesquare:nth-child(5) {
+    grid-area: 1/3/2/4;
+  }
+
+  .languagesquare:first-child,
+  .languagesquare:nth-child(2),
+  .languagesquare:nth-child(3) {
+    margin-top: 20vh;
+  }
+
+  .listaboutlanguages {
+    margin-top: 0;
+    align-items: flex-start;
+  }
+
+  .languages,
+  .frameworks {
+    justify-self: start;
+  }
+
+  .jsframeworks {
+    margin-top: 5vh;
+  }
+
+  .skills {
+    margin-top: 5vh;
+    justify-self: start;
+  }
+
+  /* -------------STARTS CONTACTME DESKTOP -------------- */
+
+  .division-line4 {
+    display: none;
+  }
+
+  .contactme {
+    display: grid;
+    grid-template-columns: 43% 52%;
+    width: 85%;
+    margin-left: 7.5%;
+  }
+
+  .always {
+    font-size: 40px;
+    line-height: 52px;
+    text-align: left;
+    margin-bottom: 30vh;
+  }
+
+  .contactform {
+    margin-top: 10vh;
+  }
+
+  .footer .division-line3 {
+    width: 100%;
+    margin-bottom: 3vh;
+  }
+
+  .container-touch {
+    justify-content: left;
+  }
+
+  .contactme .contactform .division {
+    width: 100vw;
+    margin-left: 0;
+  }
```

Comment:
Great job so far with your desktop version. 👍🏿 There is one tiny change however I would suggest you make. ♻️ 
Currently, your page extends beyond the normal width of the screens. 
![divider too long](https://github.com/eamador1/portfolio-project/assets/117144028/fbf0de5f-3a6b-4fcb-950f-e9a47c34b622)

- [ ] I suggest that you check the width above and adjust it to fit the page. ↔️